lib/screens/home.dart

```diff
@@ -0,0 +1,35 @@
+
+import 'package:flutter/material.dart';
+
+import '../models/listitem.dart';
+
+class MyApp extends StatelessWidget {
+  final List<ListItem> items;
+
+  const MyApp({Key? key, required this.items}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    const title = 'Mixed List';
+
+    return MaterialApp(
+      title: title,
+      home: Scaffold(
+        appBar: AppBar(
+          title: const Text(title),
+        ),
+        body: ListView.builder(
+          itemCount: items.length,
+          itemBuilder: (context, index) {
+            final item = items[index];
+
+            return ListTile(
+              visualDensity: VisualDensity(horizontal: 0, vertical: -4),
+              title:item.buildBody(context),
+            );
+          },
+        ),
```

Comment:
And you can answer to them or resolve them. Once all my comments are answered sufficiently, I can approve the pull request and you can merge. It's a good way to have a productive feedback loop.

Reply:
Okay. Line by line confirmed